pkg/statuschecker/types/built_in.go

```diff
@@ -108,7 +111,10 @@ func isHorizontalPodAutoscalerReady(obj runtime.Object) (isReady, retriableError
 				foundAbleToScale = true
 			case core_v1.ConditionFalse:
 				// AbleToScale should not be false if the HPA is working, this is a (retriable) failure
-				return false, true, errors.Errorf("%s: %s", cond.Reason, cond.Message)
+				return statuschecker.ObjectStatusError{
```

Comment:
Nice! Not it is just business as usual, not an error.

Reply:
Worth returning reason and message as separate fields? Both here and in other places... Not sure about this. Can be done later.